tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -29,7 +29,8 @@ export enum LeftToolbarButton {
 export class Ide {
     public static readonly EXPLORER_BUTTON_ID: string = 'shell-tab-explorer-view-container';
     public static readonly SELECTED_EXPLORER_BUTTON_CSS: string = 'li#shell-tab-explorer-view-container.theia-mod-active';
-    public static readonly ACTIVATED_IDE_IFRAME_CSS: string = '#ide-iframe-window[aria-hidden=\'false\']';
+    public static readonly THEIA_EDITOR_CSS: string = '#theia-main-content-panel';
+    // public static readonly ACTIVATED_IDE_IFRAME_CSS: string = '#ide-iframe-window[aria-hidden=\'false\']';
```

Comment:
Let's remove `ACTIVATED_IDE_IFRAME_CSS` if we don't need it anymore.

Reply:
let's remove commented code to keep script clean